book/src/math.md

```diff
@@ -0,0 +1,10 @@
+# Math
+
+Amethyst uses [nalgebra][na] under the hood as its math library and it's
```

Comment:
```suggestion
Amethyst uses [nalgebra][na] under the hood as its math library and it is
```

Reply:
May I ask why? We're using informal language all throughout the book and I have more in this paragraph alone. Why are you picking up on this particular instance?